pages/common/gettext.md

```diff
@@ -1,12 +1,22 @@
 # gettext
 
-> Get string translations.
+> Translates a string using stored translations in a compiled `.mo` file.
+> Translations are stored in `/usr/share/locale/<locale_name>/LC_MESSAGES/` with `domain` being the filename without its extension.
+> See also: `msgfmt`, `msgunfmt`.
 > More information: <https://www.gnu.org/software/gettext/manual/html_node/gettext-Invocation.html>.
 
-- Get the translation of a string or output a default string if it doesn't exist:
+- Get the translation of a string as specified in the domain file (falls back to given `msgid` if no translation exists):
 
-`LANGUAGE={{locale}} gettext {{msgid}} {{default_value}}`
+`LANGUAGE={{locale}} gettext {{[-d|--domain]}} {{domain}} "{{msgid}}"`
+
+- Suppress the trailing newline gettext generates in output:
+
+`LANGUAGE={{locale}} gettext -n {{[-d|--domain]}} {{domain}} "{{msgid}}"`
```

Comment:
Now that I'm testing this, I'm not seeing any meaningful difference. Has this been made the default or is it just on Arch? Is there a platform where you can make these commands behave differently?
![image](https://github.com/user-attachments/assets/bde4dcc8-a501-4871-a533-16a8e77b78a7)

Reply:
I'll just remove this so that we can get a page with working examples out